managers/tests/test_unit_managers_apiview.py

```diff
@@ -36,6 +36,7 @@ def setUp(self):
         self.project.members.add(self.user)
         self.url = [
             reverse('custom-projects-list'),
+            reverse('custom-project-detail', args=(self.project.pk,)),
```

Comment:
Please keep it as string, and overwrite in subclass `setUp()` method.

Reply:
Or change it into separate variables in `setUp()` like I said here: https://github.com/Code-Poets/sheetstorm/pull/85#discussion_r264261234  Choice is yours :)

Reply:
Done